src/core/push/index.ts

```diff
@@ -99,6 +99,22 @@ export function trackOpenFromQuery(ctx: Context) {
     });
 }
 
+export async function registerServiceWorker(workerPath: string) {
```

Comment:
Just moving this from utils because we only use it for push concerns?

Reply:
Yes